packages/amplication-data-service-generator/src/index.ts

```diff
@@ -1,2 +1,50 @@
-export { createDataService } from "./create-data-service";
+import { join, dirname } from "path";
+import { mkdir, readFile, writeFile } from "fs/promises";
+
+import { createDataService } from "./create-data-service";
+import { BuildContext, Module } from "./types";
+
+const [, , source, destination] = process.argv;
+if (!source) {
+  throw new Error("INPUT is not defined");
+}
+if (!destination) {
+  throw new Error("OUTPUT is not defined");
```

Comment:
There is some inconsistency here:
- We call the variable `source` but we use the term `INPUT` in the error message
- We call the variable `destination` but we use the term `OUTPUT` in the error message

Let's align on terms and just choose one. I'd vote for `source` and `destination`.

Reply:
Done